src/middlewared/middlewared/plugins/system.py

```diff
@@ -1002,6 +1002,14 @@ def _get_index(country_columns, column):
 
         return await self.config()
 
+    @accepts()
+    async def ui_restart(self):
+        """
+        Restart HTTP server to show latest UI settings.
```

Comment:
to show seems wrong here. `to use` maybe? There is nothing to show

Reply:
@freenas/docs-team please advise

Reply:
And i agree @william-gr

Reply:
`to use` works.